packages/teams-js/src/internal/communication.ts

```diff
@@ -611,42 +621,49 @@ const handleParentMessageLogger = communicationLogger.extend('handleParentMessag
 function handleParentMessage(evt: DOMMessageEvent): void {
   const logger = handleParentMessageLogger;
 
-  if ('id' in evt.data && typeof evt.data.id === 'number') {
+  if ('id' in evt.data && (typeof evt.data.id === 'number' || typeof evt.data.id === 'string')) {
```

Comment:
Seems like here (and in the other repo) we should have code that verifies that the uuid is an actual uuid of the correct format. Definitely needs to be a string to be transferred between the host and app, but we should code defensively and validate that format. Would also be good, if possible, to convert the string to a "real" UUID type of some sort after validation

Reply:
Aah you're right, I had it going through the `validateUUID` function, I forgot to push that change back up. Let me do that now!